transmart-server/build.gradle

```diff
@@ -0,0 +1,67 @@
+version "17.1"
+group "transmart.server"
```

Comment:
version, group and most of the plugins should be set by the root `build.gradle`, not in the subproject build file.